quickstarts/redisenterprise/config.yml

```diff
@@ -0,0 +1,87 @@
+# Name of the quickstart (required)
+slug: Redis Enterprise  
+
+# Displayed in the UI (required)
+title: RedisEnterprise
+
+# Long-form description of the quickstart (required)
+description: |+
+  ## RedisEnterprise Quickstart
+
+  ### RedisEnterprise is an enterprise offering of Redis that provides:
+  - High availability: Up to 5 9's
+  - Support: 24/7 Enterprise support
+  - Geo-replication: Write data consistency across multiple data centers
+  - Tiered storage: Store data on devices outside of RAM for cost savings
+  - High performance: Serve up to 200M operations per second
+
+  This integration supports [Redis Enterprise Software](https://redis.com/redis-enterprise-software/overview/) installations.
+
+  ### With the Redis Enterprise quickstart you can:
+
+  - Monitor real-time performance and trends of you Redis Enterprise cluster
+  - Drill down into the performance of individual Redis databases
+  - Understand the Redis Active/Active metrics
+
+  ### Cluster level metrics
+
+  - License Status: Information on the Enterprise license status
+  - License Capacity: Capacity licensed
+  - Used Capacity: Capacity currently in use
+  - Cluster Capacity: Information on memory and compute resources available on the cluster
+  - Total Requests: Overall throughput of the cluster in total
+
+
+  ### Database level metrics
+
+  - Database Latency: Metrics pertaining to the latency of operations at the database level
+  - Database Throughput: Detailed information of the number of read/write operations per second
+  - Database Capacity: Memory usage details for capacity planning
+  - Network Usage: Information on the network usage per database
+  - Redis Data Retention: Expiration and Eviction statistics on object retention
+
+
+  ### Active/Active metrics
+
+  - Lag: Time between application on servers
+  - Pending:  Details of the number of writes pending as part of the sync
+  - Bandwidth:  Information on the amount of traffic between clusters 
+  
+  ### Requirements
```

Comment:
```suggestion
 
```

Reply:
I don't think this section is required. The data sources will show this link prominently as a tile, and you can add another link here from the install file.